### PR TITLE
Feature/syal 930

### PR DESCRIPTION
**Pull request by HieuLM-AVISPL**
Implement talkback and its UT

### CODE REVIEW

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/talkback/controllingmetric/TalkBackSwitchOnOffControl.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	public static TalkBackSwitchOnOffControl getByApiName(String name) {
+		Optional<TalkBackSwitchOnOffControl> state = Arrays.stream(TalkBackSwitchOnOffControl.values()).filter(com -> com.getApiConfigName().equals(name)).findFirst();
+		return state.orElse(TalkBackSwitchOnOffControl.OFF);
+	}
+
+	/**
+	 * This method is used to get switch on/off code
+	 *
+	 * @param code is the code of switch on/off mode that want to get
+	 * @return State is the switch on/off mode that want to get
```
Comment (ThanhD-AVISPL) [inline src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/talkback/controllingmetric/TalkBackSwitchOnOffControl.java]: return State ... => return TalkBackSwitchOnOffControl  ...

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/dto/talkback/TalkBackStats.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TalkBackStats {
+
+	@JsonAlias("State")
+	private String state;
+
+	@JsonAlias("DestinationAddress")
+	private String destinationAddress;
+
+	public TalkBackStats() {
```
Comment (ThanhD-AVISPL) [inline src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/dto/talkback/TalkBackStats.java]: Add javadoc

File: src/test/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicatorTest.java [hunk 3]
```diff
@@ -94,6 +92,8 @@ void testHaivisionX4DecoderCommunicatorGetMonitoringDataSuccessful() {
 	@Tag("RealDevice")
 	@Test
 	void testSetDecoderSDIPrimaryStream() {
+		String configManagement = "true";
```
Comment (ThanhD-AVISPL) [inline src/test/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicatorTest.java:95]: Move configManagement = true to constant inside this class.

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicator.java [hunk 4]
```diff
@@ -308,9 +347,12 @@ public List<Statistics> getMultipleStatistics() {
 				if (cachedHDMIConfig == null) {
 					cachedHDMIConfig = new HDMIConfig(realtimeHDMIConfig);
 				}
+				if (cachedTalkbackConfig == null) {
+					cachedTalkbackConfig = new TalkBackConfig(realtimeTalkbackConfig);
+				}
 				// check Role is Admin or Operator
 				String role = retrieveUserRole();
-				if (role.equals(DecoderConstant.OPERATOR_ROLE) || role.equals(DecoderConstant.ADMIN_ROLE)) {
+				if ((role.equals(DecoderConstant.OPERATOR_ROLE) || role.equals(DecoderConstant.ADMIN_ROLE)) && validateConfigManagement()) {
```
Comment (ThanhD-AVISPL) [inline src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicator.java]: Rename method "validateConfigManagement" to "isValidConfigManagement"

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/talkback/controllingmetric/TalkBackSwitchOnOffControl.java (hunk 1 above)
Comment (ThanhD-AVISPL) [inline src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/talkback/controllingmetric/TalkBackSwitchOnOffControl.java]: return State ... => return TalkBackSwitchOnOffControl

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicator.java (hunk 4 above)
Comment (ThanhD-AVISPL) [reply src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicator.java]: line 1000: validatePortRange => isValidPortRange

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/talkback/controllingmetric/TalkBackSwitchOnOffControl.java (hunk 1 above)
Comment (HieuLM-AVISPL) [reply src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/talkback/controllingmetric/TalkBackSwitchOnOffControl.java]: I fixed it

File: src/test/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicatorTest.java (hunk 3 above)
Comment (HieuLM-AVISPL) [reply src/test/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicatorTest.java:95]: I fixed it

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/talkback/controllingmetric/TalkBackSwitchOnOffControl.java (hunk 1 above)
Comment (HieuLM-AVISPL) [reply src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/talkback/controllingmetric/TalkBackSwitchOnOffControl.java]: I fixed it

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/dto/talkback/TalkBackStats.java (hunk 2 above)
Comment (HieuLM-AVISPL) [reply src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/dto/talkback/TalkBackStats.java]: I fixed it

File: src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicator.java (hunk 4 above)
Comment (HieuLM-AVISPL) [reply src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/HaivisionXDecoderCommunicator.java]: I fixed it

**Pull request merged**